pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -222,3 +222,17 @@ func hasPathPrefix(s, pathPrefix string) bool {
 
 	return false
 }
+
+// EqualStringSlices compares string slices for equality. Slices are equal when
+// their sizes and elements on similar positions are equal.
+func EqualStringSlices(a, b []string) bool {
```

Comment:
Is this faster than `reflect.DeepEqual`? Otherwise, just use DeepEqual.

Reply:
I believe that yes, it's faster. Also this is what @brendanburns and @smarterclayton asked me to do earlier: https://github.com/kubernetes/kubernetes/pull/52799#discussion_r146095101